docroot/themes/humsci/humsci_basic/Gruntfile.js

```diff
@@ -0,0 +1,84 @@
+const sass = require('node-sass');
+const glob = require('glob');
+
+module.exports = function (grunt) {
+
+  function getIncludeFiles() {
+    const patterns = [
+      '**/*.scss',
+      'node_modules/decanter/scss',
```

Comment:
You'll likely want to include a path to the node_modules directory as well. Future releases of Decanter and Stanford Basic are going to standardize on full paths relative to the node_modules directory for anything brought in through NPM.

Reply:
Great! Thank you @sherakama I think Nathan implemented these changes in his new PR here: https://github.com/SU-HSDO/suhumsci/pull/439